a10_neutron_lbaas/v1/handler_member.py

```diff
@@ -35,14 +36,17 @@ def _create(self, c, context, member):
         server_ip = self.neutron.member_get_ip(context, member,
                                                c.device_cfg['use_float'])
         server_name = self._meta_name(member, server_ip)
-
+        conn_limit = c.device_cfg['conn-limit']
```

Comment:
There is a default value specified that covers this (I had forgotten to add it to the previous commit). Worse case, it hits the exception and fails outright.